README.md

```diff
@@ -23,3 +23,30 @@ Tip:Dont use the -nographic option on qemu when booting u-root in qemu.
 ### Roadmap : Systems to be added
 - Ubuntu
 - ArchLinux
+
+### Building a kernel for webboot
+
+webboot uses a standard kernel which should be fairly portable. This
+kernel is also based on a Long Term Stable (LTS) release. It has
+worked on every chromebook we've tried.
+
+This kernel is built using a config originally from NiChromeOS (github.com/NiChrome/NiChrome).
+Because we are building a standard boot stick formatted with vfat, we don't have the weird
+space constraints of NiChrome, so we expect this to diverge over time.
+
+Nevertheless, to keep it all simple, we build it as a non-modular kernel with the wifi
+firmware built in. We no longer build the initrd into the kernel, however; that's not needed.
+
+To build, first, be sure you're in a directory you want to be in!
+You can actually do the work right here in webboot; the .gitignore
+will ignore the two directories you create.
+
+```
+git clone --depth 1 -b v4.12.7 git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git linux
+git clone git://git.kernel.org/pub/scm/linux/kernel/git/iwlwifi/linux-firmware.git
+cp config-4.12.7 linux/.config
+(cd linux && make bzImage)
```

Comment:
when we 'make -j12' the kernel, you get an error. It does not build the kernel for you because the error is having a hard time finding openssl/bio.h. If one gets an error such as this" "scripts/extract-cert.c:21:10: fatal error: openssl/bio.h: No such file or directory
#include <openssl/bio.h>", we should put another direction below this comment to let them know to download libssl-dev, so they can build the kernel without errors. What do you think? :)

Reply:
since the installation needs permission, we should tell them to be root, but we are using this line to download libssl-dev: "apt-get install libssl-dev"

Reply:
since the installation needs permission, we are using this line to download libssl-dev: "sudo apt-get install libssl-dev"

Reply:
good catch. I'll add that.

Reply:
you can just do 
sudo apt install libssl-dev
you don't need to say apt-get any more.

Reply:
When we make the kernel, we still run into an issue. The error is "Unsupported relocation type: R_X86_64_PLT32 (4)
make[2]: *** [arch/x86/boot/compressed/Makefile:118: arch/x86/boot/compressed/vmlinux.relocs] Error 1
make[1]: *** [arch/x86/boot/Makefile:111: arch/x86/boot/compressed/vmlinux] Error 2
make: *** [arch/x86/Makefile:281: bzImage] Error 2"